plugins/dotnet/dotnet.plugin.sh

```diff
@@ -0,0 +1,54 @@
+#! bash oh-my-bash.module
+#
+# This plugin is based on the following version of the Oh-My-Zsh plugin:
+# https://github.com/ohmyzsh/ohmyzsh/blame/c68ff8aeedc2b779ae42d745457ecd443e22e212/plugins/dotnet/dotnet.plugin.zsh
+# bash parameter completion for the dotnet CLI was taken from the following
```

Comment:
```suggestion
# https://github.com/ohmyzsh/ohmyzsh/blame/c68ff8aeedc2b779ae42d745457ecd443e22e212/plugins/dotnet/dotnet.plugin.zsh

# bash parameter completion for the dotnet CLI was taken from the following
```

Sorry, could you add an empty line here? The handling of an empty line by the GitHub suggestion UI seems to be broken...